catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -463,7 +463,10 @@
                             *
                         </Import-Package>
                         <Export-Package>
-                            org.codice.ddf.catalog.ui.metacard.internal
+                            org.codice.ddf.catalog.ui.metacard.internal,
+                            org.codice.ddf.catalog.ui.api,
+                            org.codice.ddf.catalog.ui.query.cql,
+                            org.codice.ddf.catalog.ui.metacard.transform,
```

Comment:
Is there maybe a better spot to put these, like `catalog-ui-search-api`?

Reply:
Not quite sure what you mean, are you referring to this https://github.com/codice/ddf/pull/5496/files#r339094302

Reply:
Oh, ok yeah I was trying to make the same point as Ben I guess. I won't hold it up if @bdeining thinks it's okay, but would be good to at least make a ticket. 

Also where is the `org.codice.ddf.catalog.ui.metacard.transform` package? Did you mean `org.codice.ddf.catalog.ui.metacard.transformer`?

Reply:
You don't have to pull them out -- but be weary if catalog-ui-search changes  it is much more likely to to break the downstream project. One of the big benefits for having API's (the contract) and Impl's is so we can iterate and change the implementation but uphold the api that people depend on. When we directly couple to the impl (to extend, use, etc) then it makes the system brittle and easy to break. 

Having things in an API module will inform people that they need to be more aware of how changes to it could possibly impact people using that api, where as in an impl implies none of those things.

Reply:
We can split it out in a follow on ticket